packages/new-polymath-issuer/src/pages/DividendsWizard/Step-3/index.tsx

```diff
@@ -343,14 +343,34 @@ const Step3Base: FC<Props> = ({
                     component={CurrencySelect}
                     inputProps={{
                       options: [
-                        types.Tokens.Erc20,
-                        types.Tokens.Dai,
-                        types.Tokens.Gusd,
-                        types.Tokens.Pax,
-                        types.Tokens.Poly,
-                        types.Tokens.Usdc,
-                        types.Tokens.Usdt,
+                        {
+                          value: types.Tokens.Erc20,
+                        },
+                        {
+                          value: types.Tokens.Dai,
+                        },
+                        {
+                          value: types.Tokens.Gusd,
+                          isDisabled: networkId === constants.NetworkIds.Kovan,
```

Comment:
This should be stored into a variable instead of doing the comparison in every item

Reply:
Used the already existing `isTestNet` instead of this.